app/views/support_interface/application_forms/right_to_work_or_study/edit.html.erb

```diff
@@ -10,22 +10,8 @@
         <%= f.govuk_radio_button :right_to_work_or_study, 'yes', label: { text: 'Yes – I have the right to work or study in the UK' }, link_errors: true do %>
           <%= f.govuk_text_area :right_to_work_or_study_details, label: { text: 'Give details' }, hint: { text: 'For example, you have settled status or a permanent residence card' } %>
         <% end %>
-        <%= f.govuk_radio_button :right_to_work_or_study, 'no', label: { text: 'Not yet – I will need to apply for permission to work or study in the UK' } do %>
-          <p class="govuk-body">
-            Visit <%= govuk_link_to 'Get Into Teaching', 'https://beta-getintoteaching.education.gov.uk/international-candidates' %> for guidance on immigration.
-          </p>
-          <p class="govuk-body">
-            You can also <%= govuk_link_to 'register to talk to an adviser', 'https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher' %> if you need help.
-          </p>
-        <% end %>
-        <%= f.govuk_radio_button :right_to_work_or_study, 'decide_later', label: { text: 'I do not know' } do %>
-          <p class="govuk-body">
-            Visit <%= govuk_link_to 'Get Into Teaching', 'https://beta-getintoteaching.education.gov.uk/international-candidates' %> for guidance on immigration.
-          </p>
-          <p class="govuk-body">
-            You can also <%= govuk_link_to 'register to talk to an adviser', 'https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher' %> if you need help.
-          </p>
-        <% end %>
```

Comment:
Note: have removed GIT content from this form as it’s seen by support agents and this guidance is intended for candidates. Leaving it here means it’ll probably fall out of sync with what’s shown in the candidate interface.